### PR TITLE
Camel 12619

### PR DESCRIPTION
**Pull request by ffang**

### CODE REVIEW

File: components/camel-openapi-java/pom.xml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${commons-lang3-version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.apicurio</groupId>
+            <artifactId>apicurio-data-models</artifactId>
+            <version>1.0.16.Final</version>
```
Comment (oscerd) [inline components/camel-openapi-java/pom.xml]: This could be in parent pom

File: components/camel-openapi-java/pom.xml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        </dependency>
+
+
+        <!-- servlet api -->
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>2.5</version>
```
Comment (oscerd) [inline components/camel-openapi-java/pom.xml]: I think we could use a newest version?

**Comment by ffang** [comment]
Hi @oscerd ,

Thanks for your feedback!
Please see the new commit, all your request was resolved except the OSGi related. So far the apicurio isn't the OSGi bundle yet, we may do it upsteam or OSGi-fy it in Servicemix, but not sure we really want OSGi support for this component at this stage.

Cheers
Freeman

**Comment by oscerd** [comment]
> Hi @oscerd ,
> 
> Thanks for your feedback!
> Please see the new commit, all your request was resolved except the OSGi related. So far the apicurio isn't the OSGi bundle yet, we may do it upsteam or OSGi-fy it in Servicemix, but not sure we really want OSGi support for this component at this stage.
> 
> Cheers
> Freeman

Ok, so we can skip the OSGi part for the moment, a Spring Boot itest would be great

**Comment by ffang** [comment]
> Spring Boot itest

Spring Boot itest  was added also with latest commit 
690aafd

**Comment by oscerd** [comment]
> > Spring Boot itest
> 
> Spring Boot itest was added also with latest commit
> [690aafd](https://github.com/apache/camel/commit/690aafda831a7aba306bc2361032927675834d1c)

Totally missed it, thanks!

**Comment by davsclaus** [comment]
Oh the spring-boot has been moved out, can you update this PR and do a spring-boot PR against
https://github.com/apache/camel-spring-boot

**Comment by ffang** [comment]
> Oh the spring-boot has been moved out, can you update this PR and do a spring-boot PR against
> https://github.com/apache/camel-spring-boot

Sure thing, will do!

**Pull request merged**